.gitlab-ci.yml

```diff
@@ -95,22 +96,150 @@ whois-integration-test:
     - mariadb:10.2
   tags:
     - whois
+  stage: integration-test
+  rules:
+    - if: '$CI_COMMIT_REF_NAME == "master"'
+      when: always
+    - when: manual
```

Comment:
Do I need to click on run 7 times now, since we separated them? It will be quite irritating and we might forget or miss one of IT tests